client/src/Components/AboutMe/AboutMe.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./AboutMe.css";
+
+{
+  /* Two Flex box containers, about half n half width of page*/
+}
+
+function AboutMe() {
+  return (
+    <div id="AboutPage">
+      <div className="profileImageLeft">
+        <img src="../Images/IMG_1025.jpg" className="headShot"></img>
+      </div>
+      <div className="AboutParagraph">
+        <h2 id="About">About Me</h2>
+        <br></br>
+        <h4 id="Introduction">
+          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Born and raised in
+          Vermont, I grew up with a dynamic mix of being an outdoor-kid who
+          wasn't afraid to wittle away some hours playing video games or Magic
+          the Gathering. I attended Westminster College in Salt Lake City where
+          I received a B.A. in Environmental Studies. I grew to have a passion
+          for the Culinary Arts, which is what I focused on for 6 years
+          post-graduation. Working my way up the ladder in various fine-dining
+          establishments in Utah, Montana, and Vermont, my perseverence and work
+          ethic was greatly enriched.
+          <br></br>
+          {/* continuing of about me paragraphs/interests/story*/}
+           {/* PAdding is weird, cant quite get this text to have margin left */}
```

Comment:
you could wrap the specific thing you're looking to add margin left to, and give that a class or id and target it that way to adjust the margin